internal/service/project/resource_project.go

```diff
@@ -68,6 +68,25 @@ var aivenProjectSchema = map[string]*schema.Schema{
 		Description:      schemautil.Complex("The id of the billing group that is linked to this project.").Referenced().Build(),
 		DiffSuppressFunc: schemautil.EmptyObjectDiffSuppressFunc,
 	},
+	"tag": {
```

Comment:
Why is it `tag` and not `tags`? It looks confusing below with `d.Set("tag", tags)`

Reply:
This is why, a user will be adding tag one be one 
```
                  tag {
		    key   = "test"
		    value = "val"
		  }
		  tag {
		    key   = "test"
		    value = "val2"
		  }
```